components/molecules/table/components/table.js

```diff
@@ -195,7 +195,7 @@ export const Table = ({
 
 const StyledWrapper = styled.div`
   width: 100%;
-  ${({ fullHeight, isLoading }) => (fullHeight || isLoading !== undefined) && 'height: 100%'};
+  ${({ fullHeight, isLoading }) => (fullHeight || isLoading !== undefined) && 'height: 100%'}
```

Comment:
@majid-amiri I think the semi colon should be inside the quotes ?

'height: 100%;'

Reply:
Yes - If it's not outside, then should be inside.